networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -307,3 +308,41 @@ def _asr_do_remove_floating_ip(self, ri, floating_ip,
         self._edit_running_config(confstr,
                                   'REMOVE_STATIC_SRC_TRL_NO_VRF_MATCH')
         self._asr_do_remove_secondary_ip(ri, floating_ip, ex_gw_port)
+
+    def _do_set_snat_pool(self, pool_name, pool_start,
+                          pool_end, pool_net, is_delete):
+        try:
+            if is_delete:
+                conf_str = asr1k_snippets.DELETE_NAT_POOL % (
+                    pool_name, pool_start, pool_end, pool_net.netmask)
+                # TODO(update so that hosting device name is passed down)
+                self._edit_running_config(conf_str, 'DELETE_NAT_POOL')
+
+            else:
+                conf_str = asr1k_snippets.CREATE_NAT_POOL % (
+                    pool_name, pool_start, pool_end, pool_net.netmask)
+                # TODO(update so that hosting device name is passed down)
```

Comment:
does this comment still apply? Why do we have to pass the hosting device name down?